docs/platform/swarmcluster.md

```diff
@@ -179,6 +179,8 @@ The supported configuration requires the following settings:
 -   Docker Install Url: `https://platform-manager-legacy.smartsdk.eu/install-docker/17.12-smartsdk.sh`
 -   Storage Driver: `overlay2`
 -   Docker Engine Options: key: `mtu`, value `1400`
+-   Network: `node-int-net-01`
+-   FIP Pool: `federation-ext-net-01`
```

Comment:
This is the one documented in the deliverable, but I think this should be `public-ext-net-01`. Shouldn't it @dancn ?